### PR TITLE
Upgrade ChromeDriver to v2.14

### PR DESCRIPTION
**Pull request by zchee**

### CODE REVIEW

File: Library/Formula/chromedriver.rb [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 class Chromedriver < Formula
   homepage 'https://sites.google.com/a/chromium.org/chromedriver/'
-  url 'http://chromedriver.storage.googleapis.com/2.13/chromedriver_mac32.zip'
-  sha256 '0e9e9fd10b2f9b563d6b76bffe0e5222134f14423db21e6cf7fb9bd4c6a91b30'
-  version '2.13'
+  version '2.14'
+  url "http://chromedriver.storage.googleapis.com/#{version}/chromedriver_mac32.zip"
```
Comment (DomT4) [inline Library/Formula/chromedriver.rb]: Homebrew/Homebrew doesn't really use the ` #{version} ` syntax for urls. One of the maintainers may ask you to change that prior to merge.

**Comment by zchee** [comment]
OK.

I'll do fix later.

**Comment by zchee** [comment]
Fixed it.

**Comment by DomT4** [comment]
Thanks!

**Comment by zchee** [comment]
and fixed semicolon to colon

**Comment by DomT4** [comment]
Yup, Bot had an unrelated crash this morning, but you're green now :+1:

**Pull request closed without merging**